scrapers/covid-19/govScrapers/getIsrael.js

```diff
@@ -21,30 +22,43 @@ const params = {
  */
 const parseData = (data) => {
 	const dateRegex = /\d+-\d+-\d+/g;
-	const patientsPerDay = data[2].data;
-	const deadPatientsPerDay = data[3].data;
-	const recoveredPerDay = data[4].data.splice(20, data[4].data.length - 1);
-	const testsPerDay = data[5].data.splice(30, data[5].data.length - 1);
+	const patientsPerDay = data[1].data;
+	const deadPatientsPerDay = data[2].data;
+	const recoveredPerDay = data[3].data.splice(20, data[3].data.length - 1);
+	const testsPerDay = data[4].data.splice(30, data[4].data.length - 1);
```

Comment:
Add a line below this for a variable to hold doctorData: `const doctorData = data[6].data[0]`